### ISSUE TITLE
Selenium  TIMEOUT

### ISSUE THREAD

**Issue by Bernardoow** [SeleniumHQ/docker-selenium#264]
Hi Guys !!

Maybe it can be a noob question. Sorry :X

I'm  a begginer with docker.. 
I'm  using Selenium  a few months ago. And I created a crawler with it. The Crawler use selenium for get information. I Open the browser and keep it runing for some days (without docker).

But when I put on docker,  its running one time and throw exception **was terminated due to TIMEOUT.**

I searched on issues and find explation like that https://github.com/mozilla/moz-grid-config/blob/master/hub_configuration.json#L2. I use docker-composer and don't know how put composer.

Somebody can sugest me something?

Composer FILE:

selenium_hub:
  image: selenium/hub
  ports:
    - 4444:4444

node_2:
  image: selenium/node-firefox
  links:
    - selenium_hub:hub

crawler:
  restart: always
  build: ./crawler
  env_file: .env
  links:
  - postgres:postgres
  - node_2:node_2
  # - chrome:chrome
  command: python /usr/src/app/crawlerv2.py

**Comment by QAutomatron** [comment]
Hi @Bernardoow 
You should set custom $GRID_TIMEOUT for your Hub. By default it will wait for 30000 seconds and kill session and browser if no commands passed to selenium. So you should just set it to 0

I think, you selenium_hub section should looks something like this:

```
selenium_hub:
image: selenium/hub
ports:
- 4444:4444
environment:
    - GRID_TIMEOUT=0
```

**Comment by bizmate** [comment]
I am experiencing the same issue @QAutomatron since tag latest got updated to 2.53. I had to set the timeout to a really high value of 360000 to make sure selenium did not fail tests.

**Comment by ddavison** [comment]
This is a bug fixed in #288, i am in the middle of releasing right now

**Comment by Bernardoow** [comment]
@ddavison Thx MAN!
